packages/big-design/src/components/Table/HeaderCell/HeaderCell.tsx

```diff
@@ -66,29 +66,28 @@ const InternalHeaderCell = <T extends TableItem>({
     );
   };
 
-  // TODO: Update this. Return early if we don't have tooltip.
   const renderTooltip = () => {
-    if (typeof tooltip === 'string' && tooltip.length > 0) {
```

Comment:
Should we keep this check?

Reply:
@jorgemoya 
Thanks for the comment 😊.
I don't think so. I guess the checking like `if(!tooltip) { return null }` looks completely sufficient. Why do you think we need to keep the previous condition? `typeof tooltip === 'string' && tooltip.length > 0`

Reply:
It's specifically for library consumers that don't use TypeScript. It adds a layer of protection in the component and I recommend keeping it and using type-guards for other components.

Reply:
@jorgemoya @chanceaclark 👍 It makes sense. I've returned to the previous check.
Thanks)